pkg/build/controller/strategy/sti.go

```diff
@@ -39,6 +39,17 @@ func (bs *STIBuildStrategy) CreateBuildPod(build *buildapi.Build) (*kapi.Pod, er
 	if err != nil {
 		return nil, err
 	}
+
+	containerEnv := []kapi.EnvVar{
+		{Name: "BUILD", Value: string(data)},
+		{Name: "SOURCE_REPOSITORY", Value: build.Parameters.Source.Git.URI},
+	}
+
+	strategy := build.Parameters.Strategy.STIStrategy
```

Comment:
Since you are allocating ```strategy``` just to append its environment variables and never use it again, wouldn't it be more appropriate to reduce its scope eg. if strategy := build.Parameters.Strategy.STIStrategy; len(strategy.Env) > 0 {...} ?